mock/py/mockbuild/plugins/procenv.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# vim:expandtab:autoindent:tabstop=4:shiftwidth=4:filetype=python:textwidth=0:
+# License: GPL2 or later see COPYING
+
+# python library imports
+import codecs
+
+# our imports
+from mockbuild.trace_decorator import getLog, traceLog
+import mockbuild.util
+
+requires_api_version = "1.1"
+
+
+# plugin entry point
+@traceLog()
+def init(plugins, conf, buildroot):
+    ProcEnv(plugins, conf, buildroot)
+
+
+class ProcEnv(object):
+    # pylint: disable=too-few-public-methods
+    """Get the runtime process environment"""
+    @traceLog()
+    def __init__(self, plugins, conf, buildroot):
+        self.buildroot = buildroot
+        self.hw_info_opts = conf
```

Comment:
`hw_info_opts` - I guess this is left over from copy'n'paste from hw info plugin code :)

Reply:
insert embarrassed emoji.